server/src/test/java/keywhiz/service/resources/admin/SecretsResourceIntegrationTest.java

```diff
@@ -49,15 +49,18 @@
         .contains("Nobody_PgPass", "Hacking_Password", "General_Password", "NonexistentOwner_Pass",
             "Versioned_Password");
   }
+
   @Test public void listingExcludesSecretContent() throws IOException {
     // This is checking that the response body doesn't contain the secret information anywhere, not
     // just that the resulting Java objects parsed by gson don't.
     keywhizClient.login(DbSeedCommand.defaultUser, DbSeedCommand.defaultPassword.toCharArray());
 
+    String base64 = "c29tZWhvc3Quc29tZXBsYWNlLmNvbTo1NDMyOnNvbWVkYXRhYmFzZTptaXN0ZXJhd2Vzb21lOmhlbGwwTWNGbHkK";
+
     List<SanitizedSecret> sanitizedSecrets = keywhizClient.allSecrets();
     assertThat(sanitizedSecrets.toString())
-        .doesNotContain("MTMzNw==")
-        .doesNotContain(new String(Base64.getDecoder().decode("MTMzNw=="), UTF_8));
+        .doesNotContain(base64)
+        .doesNotContain(new String(Base64.getDecoder().decode(base64), UTF_8));
```

Comment:
The test was previously looking for the value "1337". I dumped all of the other non-valuable secrets from the database seed command and picked the longest one available as a replacement.